src/components/Rating/Rating.js

```diff
@@ -22,8 +22,9 @@ function CustomRating(props) {
   const { questionConfigurations } = question
 
   //[CUSTOM HOOKS]
-  const { requiredAnswerMessage } = useTranslation()
   const answersContext = useAnswersContext()
+  const { requiredAnswerMessage, checkboxIrrelevant, checkboxNonPreferred } =
```

Comment:
const { t } = useTranslations() ??

and than get translation by key

Reply:
Doesn't work in such way, needs only props drilling. Maybe needs refactoring on higher level...